processing/src/main/java/io/druid/query/groupby/epinephelinae/LimitedBufferHashGrouper.java

```diff
@@ -375,27 +374,25 @@ public int compare(Integer o1, Integer o2)
     };
   }
 
-
-  private void validateBufferCapacity(
-      int limit,
-      float maxLoadFactor,
-      ByteBuffer buffer,
-      int bucketSize
-  )
+  public boolean validateBufferCapacity(int bufferCapacity)
   {
     int numBucketsNeeded = (int) Math.ceil((limit + 1) / maxLoadFactor);
     int targetTableArenaSize = numBucketsNeeded * bucketSize * 2;
     int heapSize = (limit + 1) * (Ints.BYTES);
     int requiredSize = targetTableArenaSize + heapSize;
 
-    if (buffer.capacity() < requiredSize) {
-      throw new IAE(
-          "Buffer capacity [%d] is too small for limit[%d] with load factor[%f], minimum bytes needed: [%d]",
-          buffer.capacity(),
+    if (bufferCapacity < requiredSize) {
+      log.info(
```

Comment:
I think "debug" is better here -- this kind of diagnostic is a bit too loud for logging by default.

Reply:
changed to debug